frame/subframe.go

```diff
@@ -383,7 +383,7 @@ func (subframe *Subframe) decodeRicePart(paramSize uint) error {
 		if err != nil {
 			return unexpected(err)
 		}
-		if paramSize == 4 && x == 0xF || paramSize == 4 && x == 0x1F {
+		if paramSize == 4 && x == 0xF || paramSize == 5 && x == 0x1F {
```

Comment:
Thanks for spotting this! Do you happen to have a FLAC file which uses Rice parameter escape codes? I would like to implement this functionality but have deferred it since I don't have any FLAC file to verify the implementation against.